storage/objects/objects_test.go

```diff
@@ -173,6 +173,21 @@ func TestObjects(t *testing.T) {
 		t.Errorf("contents = %q; want %q", got, want)
 	}
 
+	t.Run("setMetadata", func(t *testing.T) {
```

Comment:
subtest because of this comment:
https://github.com/GoogleCloudPlatform/golang-samples/pull/1784#discussion_r510496871